sentinelsat/products.py

```diff
@@ -1,32 +1,10 @@
-import os
-import shutil
+from fnmatch import fnmatch
```

Comment:
The Google Style Guide [discourages](https://google.github.io/styleguide/pyguide.html#22-imports) any `from` imports of functions and classes, except for `typing`.

We do not follow the G style guide explicitly, but should we perhaps still adhere to this nice rule? That would require you to use `fnmatch.fnmatch` and `pathlib.Path` everywhere.

Reply:
I somewhat agree with that style guide suggestion, but within reason. I find the `pathlib.Path` to be such a basic and common class in Python that using it as `from pathlib import Path` seems almost idiomatic and the alternative looks clunky to me. I agree for the `fnmatch.fnmatch`, though.